Sources/Public/SCCAPIRequest.h

```diff
@@ -50,13 +50,6 @@ typedef NS_OPTIONS(NSUInteger, SCCAPIRequestTenderTypes) {
  */
 @interface SCCAPIRequest : NSObject <NSCopying>
 
-/**
- Sets the client ID to associate with all subsequent API requests.
- The client ID must be set to something other than nil before the first API request object is created.
- @param clientID The client ID to associate with all subsequent API requests.
- */
-+ (void)setClientID:(nullable NSString *)clientID __deprecated_msg("Use the favored `setApplicationID` instead.");
-
 /**
  Sets application client ID to associate with all subsequent API requests.
```

Comment:
```suggestion
 Sets application ID to associate with all subsequent API requests.
```

lol that was confusing